### PR TITLE
refactor(templates): remove unused imports

### PR DESCRIPTION
**Pull request by machi1990**

### CODE REVIEW

File: templates/ts-apollo-mongodb-backend/src/index.ts [hunk 1]
```diff
@@ -1,55 +1,54 @@
...
-    dataProviderCreator: createMongoDbProvider(db)
+    dataProviderCreator: createMongoDbProvider(db),
   });
 
   const apolloServer = new ApolloServer({
     typeDefs,
     resolvers: [resolvers, noteResolvers],
-    context: contextCreator
-  })
+    context: contextCreator,
```
Comment (craicoverflow) [inline templates/ts-apollo-mongodb-backend/src/index.ts:41]: Does your editor prefer trailing commas? Not that it matters, they are useful!

Comment (machi1990) [reply templates/ts-apollo-mongodb-backend/src/index.ts]: Ah this should not have been committed. I ran `prettier` on the wrong project.

File: templates/ts-apollo-mongodb-backend/src/index.ts [hunk 2]
```diff
@@ -1,36 +1,35 @@
 // eslint-disable-next-line @typescript-eslint/no-require-imports
-require('dotenv').config()
-import path from 'path'
-import { ApolloServer } from "apollo-server-express"
-import { buildGraphbackAPI } from 'graphback'
-import { loadSchemaSync } from '@graphql-tools/load'
-import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
```
Comment (craicoverflow) [inline templates/ts-apollo-mongodb-backend/src/index.ts:7]: These can also be removed from `package.json`

File: templates/ts-apollo-mongodb-datasync-backend/src/index.ts [hunk 3]
```diff
@@ -1,24 +1,20 @@
 // eslint-disable-next-line @typescript-eslint/no-require-imports
-require('dotenv').config()
-import path from 'path'
-import http from "http"
-import { ApolloServer } from "apollo-server-express"
-import { loadSchemaSync } from '@graphql-tools/load'
-import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
```
Comment (craicoverflow) [inline templates/ts-apollo-mongodb-datasync-backend/src/index.ts:7]: These can also be removed from `package.json`

File: templates/ts-apollo-postgres-backend/src/index.ts [hunk 4]
```diff
@@ -1,22 +1,19 @@
 // eslint-disable-next-line @typescript-eslint/no-require-imports
-require('dotenv').config()
-import { ApolloServer } from "apollo-server-express"
-import cors from "cors"
-import express from "express"
-import http from "http"
-import { buildGraphbackAPI } from 'graphback'
-import { loadSchemaSync } from '@graphql-tools/load'
-import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
```
Comment (craicoverflow) [inline templates/ts-apollo-postgres-backend/src/index.ts:9]: These can also be removed from `package.json`

**Review by craicoverflow — CHANGES_REQUESTED** [review]

File: templates/ts-apollo-mongodb-backend/src/index.ts (hunk 2 above)
Comment (machi1990) [reply templates/ts-apollo-mongodb-backend/src/index.ts:7]: Good catch.

**Review by craicoverflow — APPROVED** [review]

**Pull request merged**